.env

```diff
@@ -1,3 +1,5 @@
+#stage
 BASE_PATH=https://wxn4aqqc8bqvxcy6unfe.api.stage.openshift.com
-#BASE_PATH=https://cos-fm.apps.kammellol.rhmw-integrations.net
+#dev
+#BASE_PATH=https://cos-fleet-manager-cos.rh-fuse-153f1de160110098c1928a6c05e19444-0000.eu-de.containers.appdomain.cloud
```

Comment:
@gashcrumb you may have to test it against the dev cluster here

Reply:
:+1: yep, thanks!